app/modules.go

```diff
@@ -189,7 +189,6 @@ func appModules(
 		icq.NewAppModule(*app.AppKeepers.ICQKeeper),
 		packetforward.NewAppModule(app.PacketForwardKeeper),
 		cwpoolmodule.NewAppModule(appCodec, *app.CosmwasmPoolKeeper),
-		cwpoolmodule.NewAppModule(appCodec, *app.CosmwasmPoolKeeper),
```

Comment:
Do we need to remove this?

Reply:
yeah, it's registered twice. I think it was already removed on main